pom.xml

```diff
@@ -12,7 +12,7 @@
   <name>filing-history-data-api</name>
 
   <artifactId>filing-history-data-api</artifactId>
-  <version>latest</version>
+  <version>unversioned</version>
```

Comment:
is this by mistake ?

Reply:
That was last week. I no longer remember. I'll take it out and retest

Reply:
Dropped the L15 change